source/draw/RasterMonochrome.ooc

```diff
@@ -33,6 +33,7 @@ RasterMonochrome: class extends RasterPacked {
 	create: func (size: IntSize2D) -> Image { This new(size) }
 	copy: func -> This {
 		result := This new(this)
+		result _buffer = ByteBuffer new(this _buffer size)
```

Comment:
> Or, all subclasses to RasterPacked create the buffer in their respective Copy method (like RasterMonochrome does here).

That's exactly what I want to avoid - the code in `copy` will look 100% identical in all classes that inherit from `RasterPacked` (it already does, except for the line you just added to `RasterMonochrome` here).